localgraphclustering/SimpleLocal.py

```diff
@@ -35,6 +44,23 @@ def SimpleLocal(G, ref_nodes,
 
     """ 
     n = G.adjacency_matrix.shape[0]
-    (actual_length,actual_xids) = SimpleLocal_cpp(n,G.ai,G.aj,len(ref_nodes),ref_nodes,delta)
+    (actual_length,actual_xids) = SimpleLocal_cpp(n,G.ai,G.aj,len(ref_nodes),ref_nodes,delta,relcondflag)
+    # Use DFS to guarantee the returned set is connected
+    if relcondflag:
```

Comment:
You should probably handle this in the C++ code that give has an optional flag to return only the connected component. We can start with this code, but that's something we should do in the future.